tests/unit/pyrflx_test.py

```diff
@@ -1179,3 +1180,10 @@ def test_message_size_unverified() -> None:
     message.set("A", 2)
     message.set("B", b"\x01\x02")
     assert message.valid_message
+
+
+def test_message_last(message_type_size_value: MessageValue) -> None:
+    message_type_size_value.parse(b"\x08\x02")
+    assert message_type_size_value.valid_message
+    assert message_type_size_value.get("F1") == 8
+    assert message_type_size_value.get("F2") == 2
```

Comment:
Why is that function called `test_message_last`?

Reply:
That's a good question.
Renamed to `test_message_type_size_condition` and squashed commit it into cf80142.